File: Practica9/Python/Ejercicio 2/Circulo.py
import math
from Figura import Figura
class Circulo(Figura):
    def __init__(self, radio, color):
        super().__init__(color)
        self.radio = radio

    def area(self):
        return math.pi * self.radio * self.radio

    def perimetro(self):
        return 2 * math.pi * self.radio

    def __str__(self):
        return "Círculo - " + super().__str__() + \
               ", Área: " + repr(self.area()) + \
               ", Perímetro: " + repr(self.perimetro())

File: Practica9/Java/Ejercicio 2/Circulo.java
import java.awt.*;
class Circulo extends Figura {
    double radio;

    Circulo(double r, String c) {
        super(c);
        radio = r;
    }

    double area() {
        return Math.PI * radio * radio;
    }

    double perimetro() {
        return 2 * Math.PI * radio;
    }

    void dibujar(Graphics g, int x, int y) {
        g.setColor(Color.decode(color));
        g.fillOval(x, y, (int)(2*radio), (int)(2*radio));
    }
}